src/main/update/player-update.ts

```diff
@@ -44,7 +44,7 @@ export async function playerUpdate(
   if (fs.existsSync(playerPath)) {
     fs.rmdirSync(playerPath, { recursive: true });
   } else {
-    fs.mkdirSync(playerPath);
+    fs.mkdirSync(playerPath, { recursive: true });
```

Comment:
```suggestion
    await fs.promises.mkdir(playerPath, { recursive: true });
```

Synchronous functions block the thread while running, but I guess this is for other PRs.

Reply:
@.@ I dont know why I did it
I'll fix it other PR Thanks 😵‍💫